python/ray/actor.py

```diff
@@ -214,6 +322,10 @@ def export_actor(actor_id, class_id, actor_method_names, num_cpus, num_gpus,
 
 
 def make_actor(cls, num_cpus, num_gpus, checkpoint_interval):
+    # Add one to the checkpoint interval since we will insert a mock task for
+    # every checkpoint.
+    checkpoint_interval += 1
```

Comment:
Not a big deal, but 

```python
@ray.remote(checkpoint_interval=0)
class Foo(object):
    def __init__(self):
        pass
```

causes an infinite loop :)

Reply:
Ah yeah, this is a little unfortunate...